### PR TITLE
Configure Babel To Server

### LINKED ISSUE #4: Add babel to the server side to use ES6 features.

### PR DESCRIPTION
**Pull request by nitishmittal1990**
Closes #4

### CODE REVIEW

**Review by fenilgandhi — APPROVED** [review]

File: yarn.lock [hunk 1]
```diff
@@ -1822,6 +1849,14 @@ ansi-styles@^4.1.0:
     "@types/color-name" "^1.1.1"
     color-convert "^2.0.1"
 
+<<<<<<< HEAD
```
Comment (fenilgandhi) [inline yarn.lock]: Hey @nitishmittal1990 , need to resolve this.

File: yarn.lock [hunk 2]
```diff
@@ -3245,6 +3316,7 @@ chardet@^0.7.0:
   resolved "https://registry.yarnpkg.com/chardet/-/chardet-0.7.0.tgz#90094849f0937f2eedc2425d0d28a9e5f0cbad9e"
   integrity sha512-mT8iDcrh03qDGRRmoA2hmBJnxpllMR+0/0qlzjqZES6NdiWDcZkCNAk4rPFZ9Q85r27unkiNNg8ZOiwZXBHwcA==
 
+<<<<<<< HEAD
```
Comment (fenilgandhi) [inline yarn.lock]: and this.

File: yarn.lock [hunk 3]
```diff
@@ -3258,6 +3330,9 @@ cheerio@^1.0.0-rc.3:
     parse5 "^3.0.1"
 
 chokidar@^1.6.1:
+=======
```
Comment (fenilgandhi) [inline yarn.lock]: and this too.

File: yarn.lock [hunk 4]
```diff
@@ -3367,6 +3439,14 @@ clean-stack@^2.0.0:
   resolved "https://registry.yarnpkg.com/clean-stack/-/clean-stack-2.2.0.tgz#ee8472dbb129e727b31e8a10a427dee9dfe4008b"
   integrity sha512-4diC9HaTE+KRAMWhDhrGOECgWZxoevMc5TlkObMqNSsVU62PYzXZ/SMTjzyGAFF1YusgxGcSWTEXBhp0CPwQ1A==
 
+<<<<<<< HEAD
```
Comment (fenilgandhi) [inline yarn.lock]: .

File: yarn.lock [hunk 5]
```diff
@@ -3515,6 +3595,7 @@ combined-stream@^1.0.6, combined-stream@^1.0.8, combined-stream@~1.0.6:
   dependencies:
     delayed-stream "~1.0.0"
 
+<<<<<<< HEAD
```
Comment (fenilgandhi) [inline yarn.lock]: ..

File: yarn.lock [hunk 6]
```diff
@@ -3523,6 +3604,9 @@ commander@2.8.x:
     graceful-readlink ">= 1.0.0"
 
 commander@^2.11.0, commander@^2.19.0, commander@^2.20.0:
+=======
```
Comment (fenilgandhi) [inline yarn.lock]: ..

File: yarn.lock [hunk 7]
```diff
@@ -3604,6 +3688,21 @@ concurrently@^5.1.0:
     tree-kill "^1.2.2"
     yargs "^13.3.0"
 
+<<<<<<< HEAD
```
Comment (fenilgandhi) [inline yarn.lock]: ...

**Review by fenilgandhi — CHANGES_REQUESTED** [review]

**Review by fenilgandhi — APPROVED** [review]

**Comment by rishabh1403** [comment]
Use better PR names. 
also some files are missing new line at the end. Please resolve those

**Comment by nitishmittal1990** [comment]
> Use better PR names.
> also some files are missing new line at the end. Please resolve those

Updated above 2 points

**Review by nimish-gupta — APPROVED** [review]
LGTM.

**Pull request merged**